modules/react-components/src/components/card/selection-card.tsx

```diff
@@ -148,8 +159,18 @@ export const SelectionCard: FunctionComponent<SelectionCardPropsInterface> = (
             link={ false }
             as="div"
             data-testid={ testId }
+            onMouseEnter={ () => setDimmerState(true) }
+            onMouseLeave={ () => setDimmerState(false) }
             { ...rest }
         >
+            {
+                // TODO: This should pass as prop, where this component is used,
+                renderDisabledItemsAsGrayscale && (
```

Comment:
Should we add a new prop like `renderPreviewItemsAsDimmed` and use it here instead of `renderDisabledItemsAsGrayscale`? @inthirakumaaran @brionmario

Reply:
Yes. Let's name it as `overlay` and get the dimmer from the app side.